website/docs/r/public_ip.html.markdown

```diff
@@ -42,7 +42,7 @@ The following arguments are supported:
 
 * `allocation_method` - (Required)  Defines the allocation method for this IP address. Possible values are `Static` or `Dynamic`.
 
-~> **Note** `Dynamic` Public IP Addresses aren't allocated until they're assigned to a resource (such as a Virtual Machine or a Load Balancer) by design within Azure - [more information is available below](#ip_address).
+~> **Note** `Dynamic` Public IP Addresses aren't allocated until they're assigned to a resource (such as a Virtual Machine or a Load Balancer) by design within Azure. See `ip_address` argument.
```

Comment:
this seems like an error?

Reply:
The `#ip_address` link anchor is not a proper Markdown heading, thus markdownlint complains about it breaking `MD051/link-fragments` rule. This is the only instance in the documentation so far that had this unique way of linking to other argument.

How about the following sentence?

See `ip_address` argument below.

Reply:
works for me!